library/src/test/java/com/novoda/simplechromecustomtabs/provider/SimpleChromeCustomTabsAvailableAppProviderTest.java

```diff
@@ -110,4 +106,13 @@ private void givenThatPackageIsNotFound() {
         givenThatPackageIsEmpty();
     }
 
+    private static class StubExecutor implements Executor {
+        private final Handler mHandler = new Handler(Looper.getMainLooper());
+
+        @Override
+        public void execute(Runnable command) {
+            mHandler.post(command);
```

Comment:
as the test is run on the main thread by default you could just do `command.run()` here